WalletWasabi/WebClients/Wasabi/WasabiClientFactory.cs

```diff
@@ -41,5 +41,13 @@ public WasabiClient NewBackendClient(bool isolateStream = false)
 			TorHttpClient torHttpClient = new TorHttpClient(BackendUriGetter.Invoke(), TorEndpoint, isolateStream: isolateStream);
 			return new WasabiClient(torHttpClient);
 		}
+
+		/// <summary>
+		/// Creates new <see cref="TorHttpClient"/>.
+		/// </summary>
+		public TorHttpClient NewBackendTorHttpClient(bool isolateStream)
```

Comment:
Is this a temporary function? If not this can be used by NewBackendClient

Reply:
It is a temporary function. I have realized that `isolateStream` would be best to have in `httpClient.SendAsync` as a parameter. It would make the API much easier to use. As you would not need to instantiate new instances of TorHttpClient etc.

Reply:
ACK